model-optimizer/extensions/middle/UselessSplitEraser.py

```diff
@@ -21,13 +21,13 @@
 class UselessSplitEraser(MiddleReplacementPattern):
     enabled = True
 
-    def run_after(self):
-        from extensions.middle.pass_separator import PreMiddleStart
-        return [PreMiddleStart]
-
-    def run_before(self):
-        from extensions.middle.pass_separator import MiddleStart
-        return [MiddleStart]
+    # def run_after(self):
```

Comment:
Why did you commented these lines? Do you understand why this transformation was executed exactly between PreMiddleStart and MiddleStart?

Reply:
Because if we have the sub-graph `Split->Concat`, an input of `Split` is 4D or 5D tensor and `num_splits == 1`, inputs of `Concat` go only from `Split` and outputs of `Split` is only this `Concat`, then we can replace this pair with `Interpolate`.

Reply:
1. It is not a good practice to leave commented code. If you don't need the code then remove it.
2. In the case which you described above the `Split->Concat` pair can be simply removed.

Please, revert this change and do not introduce dependency to the `UselessSplitEraser` in the `SplitConcatPairToInterpolate `

Reply:
Fixed.